splight_lib/models/native.py

```diff
@@ -23,21 +23,24 @@ def set_output_format(cls, v):
     def get(
         cls, asset: str, attribute: str, **params: Dict
     ) -> List["NativeOutput"]:
-        params["output_format"] = cls._output_format
         return super().get(asset, attribute, **params)
 
     @classmethod
-    async def async_get(cls, **params: Dict) -> List["NativeOutput"]:
-        params["output_format"] = cls._output_format
-        return await super().async_get(**params)
+    async def async_get(
+        cls, asset: str, attribute: str, **params: Dict
+    ) -> List["NativeOutput"]:
+        return await super().async_get(asset, attribute, **params)
```

Comment:
we don't need output_format anymore?

Reply:
Nop, that value is already stored in the datalake and we are not using that field for quering the data